pkg/systemd/bootInfo.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from "react";
+import cockpit from "cockpit";
+import {
+    Card,
+    CardBody,
+    CardTitle, CodeBlock, CodeBlockCode,
+    EmptyStateHeader,
+    EmptyStateVariant,
+    List,
+    ListItem,
+    ListVariant, Spinner
+} from "@patternfly/react-core";
+import "./bootInfo.scss";
+import { EmptyState, EmptyStateBody } from "@patternfly/react-core/dist/esm/components/EmptyState";
+
+const _ = cockpit.gettext;
+
+export function BootInfo() {
+    const [svg, setSvg] = useState(undefined);
+    const [summary, setSummary] = useState(null);
+
+    useEffect(() => {
+        let string = "";
+        cockpit.spawn(["systemd-analyze", "plot"])
+                .stream((svg_part) => {
+                    string += svg_part;
+                })
+                .then(() => {
+                    const doc = new DOMParser().parseFromString(string, "text/xml");
+                    doc.querySelector("rect.background")?.remove();
+                    doc.querySelector("text[y='30']")?.remove();
+                    const svgElem = doc.querySelector("svg");
+                    svgElem.style.scale = "0.5";
+                    const summaryElem = doc.querySelector("text[y='50']");
```

Comment:
Similar thing, making assumptions about the output. But at least this one is easier to test.

How about this: You grab *all* top-level `<text>` elements and move them into the card? That's conceptually cleaner and more robust.